docs/source/features/long_context_recipe.rst

```diff
@@ -0,0 +1,108 @@
+.. _long_context_recipe:
+
+Long Context Recipe
+------------------------
+
+Long context model training enhances the capability of large language models to handle long context inputs. Longer sequence lengths are beneficial for many NLP tasks, such as document-level summarization, long document classification, and long document question answering. NeMo Framework provides a recipe to train long context models like Llama-3, Mixtral, and Nemotron.
+
+Access Long Context Recipe
+========================
+
+NeMo 2.0 is providing tested recipes to train long context models. `The recipe is available in the NeMo LLM GPT Trainer Directory <https://github.com/NVIDIA/NeMo/tree/main/nemo/collections/llm/recipes>`_ directory.
+
+The following charts for the Llama-3, Mixtral, and Nemotron models show the different sequence lengths supported by each model at various sizes.
+
+Llama-3
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table:: Llama-3
+   :widths: 30 25 25
+   :header-rows: 1
+
+   * - Sequence Length
+     - 8B
+     - 70B
+   * - 16k
+     - Yes
+     - Yes
+   * - 64k
+     - Yes
+     - Yes
+   * - 128k
+     - TODO
+     - TODO
+
+Mixtal
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table:: Mixtral
+   :widths: 30 25 25 25
+   :header-rows: 1
+
+   * - Sequence Length
+     - 8x3B
+     - 8x7B
+     - 8x22B
+   * - 16k
+     - Yes
+     - Yes
+     - TODO
+   * - 64k
+     - Yes
+     - Yes
+     - TODO
+   * - 128k
+     - TODO
+     - TODO
+     - TODO
+
+Nemotron (Not yet supported in NeMo 2.0)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table:: Nemotron
+   :widths: 30 25 25
+   :header-rows: 1
+
+   * - Sequence Length
+     - 15B
+     - 340B
+   * - 16k
+     - TODO
+     - TODO
+   * - 64k
+     - TODO
+     - TODO
+   * - 128k
+     - TODO
+     - TODO
+
+
+Context Parallelism
+========================
+
+Context Parallelism (CP) is a method for parallelizing the processing of neural network activations across multiple GPUs by partitioning the input tensors along the sequence dimension.Unlike Sequence Parallelism (SP) that partitions the activations of specific layers, CP divides the activations of all layers.
```

Comment:
add space between sentences.

Context Parallelism (CP) is a method for parallelizing the processing of neural network activations across multiple GPUs by partitioning the input tensors along the sequence dimension. Unlike Sequence Parallelism (SP) that partitions the activations of specific layers, CP divides the activations of all layers.